### ISSUE TITLE
Error loading Package Control plugin (4.0.0-beta3)

### ISSUE THREAD

**Issue by manastungare** [wbond/package_control#1644]
Plugin version: https://github.com/wbond/package_control/releases/tag/4.0.0-beta3

(I tried the Beta because the prod version also does not load on my machine, but I’m reporting issues with the beta, assuming this is the next RC).

When I turn `"debug": true`, I get in the console:

```
Unable to open /Users/manas/Library/Application Support/Sublime Text/Packages/Package Control/Package Control.sublime-settings
```

Turning off `debug`, I see logs showing the plugin failed to load:

```
reloading plugin Package Control.plugin
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python38/sublime_plugin.py", line 325, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.8/importlib/__init__.py", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 808, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/manas/Library/Application Support/Sublime Text/Installed Packages/Package Control.sublime-package/plugin.py", line 371, in <module>
  File "/Users/manas/Library/Application Support/Sublime Text/Installed Packages/Package Control.sublime-package/package_control/commands/__init__.py", line 4, in <module>
  File "/Users/manas/Library/Application Support/Sublime Text/Installed Packages/Package Control.sublime-package/package_control/commands/disable_package_command.py", line 3, in <module>
  File "/Users/manas/Library/Application Support/Sublime Text/Installed Packages/Package Control.sublime-package/package_control/package_disabler.py", line 10, in <module>
ModuleNotFoundError: No module named 'package_control'
plugins loaded
```

**Comment by deathaxe** [comment]
Thanks for reporting back. That's indeed a regression compared to beta2, which is triggered at the very first start with new Package Control 4.0.

**Comment by deathaxe** [comment]
Fixed by 4.0.0-beta4

**Comment by manastungare** [comment]
Verified with `4.0.0-beta4`, thank you so much for the quick fix, I (and many others) appreciate it!
